src/ucp/wireup/select.c

```diff
@@ -772,6 +785,10 @@ static ucs_status_t ucp_wireup_add_am_lane(ucp_ep_h ep, const ucp_ep_params_t *p
     int need_am;
     uint32_t usage;
 
+    if (ucp_is_ep_mem_type(params)) {
```

Comment:
need  to unite this with the "if" on line 792. it works now because cudacopy/gdrcopy are not p2p transports